0003-interledger-protocol/0003-interledger-protocol.md

```diff
@@ -243,10 +254,105 @@ Not all ledgers support held transfers. In the case of a ledger that doesn't, th
 2. The receiver MAY trust the sender. The sender will notify the receiver about the intent to transfer. If the receiver provides a fulfillment for the condition before the expiry date, the sender will perform a regular transfer to the receiver.
 
 3. The sender and receiver MAY appoint a mutually trusted third-party which has an account on the local ledger. The sender performs a regular transfer into a neutral third-party account. In the first step, funds are transfered into the account belonging to the neutral third-party.
-### Payment Channels
+
+### ILP Error Format
+
+Here is a summary of the fields in the ILP error format:
+
+| Field | Type | Short Description |
+|:--|:--|:--|
+| code | IA5String | [ILP Error Code](#ilp-error-codes) |
+| name | IA5String | [ILP Error Code Name](#ilp-error-codes) |
+| triggeredBy | Address | ILP address of the entity that originally emitted the error |
+| forwardedBy | SEQUENCE OF Address | ILP addresses of connectors that relayed the error message |
+| triggeredAt | Timestamp | Time when the error was initially emitted |
+| data | OCTET STRING | Error data provided for debugging purposes |
+
+#### code
+
+    IA5String (SIZE(3))
+
+Error code. For example, `F00`. See [ILP Error Codes](#ilp-error-codes) for the list of error codes and their meanings.
+
+#### name
+
+    IA5String
+
+Error name. For example, `Bad Request`. See [ILP Error Codes](#ilp-error-codes) for the list of error codes and their meanings.
+
+#### triggeredBy
+
+[ILP Address](#account) of the entity that originally emitted the error.
+
+#### forwardedBy
+
+[ILP Addresses](#account) of the connectors that relayed the error message.
```

Comment:
Added a comment about that to `emittedBy`. I don't think it would be worth having ledgers include their prefixes in the `forwardedBy` array